pkg/ingester/user_tsdb.go

```diff
@@ -289,8 +294,8 @@ func (u *userTSDB) PreCreation(metric labels.Labels) error {
 	}
 
 	// Total series limit.
-	series, shards := u.getSeriesAndShardsForSeriesLimit()
-	if !u.limiter.IsWithinMaxSeriesPerUser(u.userID, series, shards) {
+	series, minLimit := u.getSeriesAndMinForSeriesLimit()
```

Comment:
[nit] I suggest to be consistent with naming. It's called `minLocalLimit` in other places and I suggest to call it `minLocalLimit` here as well (it's clearer and keep consistency).

Reply:
changed in 82e5e8229ddce202770927c7eda43a06a502ac63